components/support/sync-telemetry/build.gradle

```diff
@@ -42,7 +42,9 @@ android {
 dependencies {
     implementation project(':support-base')
 
-    implementation project(':service-glean')
+    implementation(project(':service-glean')) {
+        exclude group: 'org.mozilla.telemetry', module: 'glean-native'
```

Comment:
So it's not entirely clear to me why this is necessary, but otherwise this was causing build problems for me with the error that glean-native comes both from glean-native and geckoview-omni.